### ISSUE TITLE
Player facing/orientation and interactions

### ISSUE THREAD

**Issue by felix123-cpu** [Nibre/MotherVR#204]
Hi, 

I believe the mod is great and perfectly playable, except for one issue:
The player is able to rotate/spin freely with a 6dof headset, but the "forward" direction stays static and doesn't get updated. This makes for disorienting transitions when interacting with terminals or other objects, since the camera gets moved based on the static/outdated "forward" position, instead of the current headset orientation.

If a fix would break too many other things, may I suggest the following:
The l1/r1 VR orientation "reset" should automatically be triggered in the same frame that the player interacts with terminals etc. Ideally, this would make for a smooth transition.

**Comment by matejdro** [comment]
Yeah this is probably the most annoying issue with the mod. I will look at an object, press interact button and my view will suddenly turn around 90 degrees.
